src/views/Profile.vue

```diff
@@ -1,32 +1,64 @@
 <template>
-  <div id="profile_page" :class="this.$store.state.css">
-    <h1>Profil utilisateur</h1>
-    <h3>Cette page est encore en construction</h3>
-    <p>Si vous avez été redirigé automatiquement ici, c'est que votre abonnement n'est pas à jour</p>
-    <p><a class="teamSelector" v-on:click="goToBillingPortal">{{$t('BILLING_MANAGEMENT')}}</a></p>
-    <NavMenu :page="'profile'" />
+  <div :class="this.$store.state.css">
+    <div class="profilpage">
+      <div class="col-12 offset-0 col-sm-8 offset-sm-2 col-md-6 offset-md-3 col-lg-4 offset-lg-4 col-xxl-3 offset-xxl-5 ">
+        <div class="image-container">
+           <img id="logoface" alt="Vue logoface" src="../assets/logo_round.png" />
+        </div>
+         <h3 class="ProfilName">Nom Prenom</h3>
```

Comment:
classes' names are in english in all the code and profile have an -e in english. Furthermore, they are written in camelCase : so line 3 profilpage become profilePage and line 8 ProfilName become profileName.
You can check all your classes and id to rectify their names and have a continuity in all code :)